src/run_common.cpp

```diff
@@ -36,6 +36,12 @@ void setup_common_run_options(
             "t is the current logical time and d is the amount of logical "
             "time that has passed since the start of the simulation.  "
             "t and d are floating-point numbers while n is an integer.")
+        ("worker-threads",
+            boost::program_options::value<int>()->value_name("worker-threads")->implicit_value(-1),
+            "Enables spawning worker-threads to parallelize the work load. "
+            "If not specified, the number of worker-threads is set to the "
```

Comment:
Regardless of whether `default_value` or `implicit_value` is used, the value -1 will be shown in the help text that is autogenerated by Boost.program_options. It might be confusing if it is not explained clearly that it's a special value. Maybe something like:
```
The default (represented by the value -1) is to use the number of system hardware cores minus one.
```